### ISSUE TITLE
你好，关于 Nextjs 服务端无法触发 saga 异步的问题

### ISSUE THREAD

**Issue by iiDestiny** [victor36max/next-saga-example#1]
大哥，我想向您请教一下，关于 Nextjs 服务端无法触发 saga 异步的问题

Nextjs + Next-redux-wrapper：在本地换中执行 `npm run dev` 构建好运用后，我在 _app.jsx 文件服务端中执行了获取用户的 dispatch， saga 并没有执行异步请求。

但是当我随意修改 jsx 文件终端热更新后，dispatch 的用户信息又能正常使用了。这令人费解。希望能得到您的指教，谢谢！

这是在 Next-redux-wrapper 官方提的 issue：https://github.com/kirill-konshin/next-redux-wrapper/issues/397

这是 codesandbox 在线例子：https://codesandbox.io/s/jovial-night-mtkbt?file=/pages/index.js

**Comment by iiDestiny** [comment]
Brother, I would like to ask you a question about the Nextjs server side not being able to trigger saga asynchronously

Nextjs + Next-redux-wrapper: After building the application by running `npm run dev` in the local swap, I executed the dispatch to get the user in the _app.jsx file server side and saga did not execute the asynchronous request.

But when I modified the jsx file terminal hot update at random, the dispatch's user information worked again. This is puzzling. I'd like to ask for your advice, thanks!

This is the official issue raised in next-redux-wrapper: https://github.com/kirill-konshin/next-redux-wrapper/issues/397

This is the codesandbox online example: https://codesandbox.io/s/jovial-night-mtkbt?file=/pages/index.js
